Xargo.toml

```diff
@@ -0,0 +1,5 @@
+[target.x86_64-uwp-windows-msvc.dependencies]
+std = { features = ["panic-unwind"] }
+
+[target.aarch64-uwp-windows-msvc.dependencies]
+std = { features = ["panic-unwind"] }
```

Comment:
This appears to be undoing https://github.com/rust-lang/rust/blob/000d90b11f7be70ffb7812680f7abc6deb52ec88/src/librustc_target/spec/aarch64_uwp_windows_msvc.rs#L8-L9 , which has a comment that says it shouldn’t be there. Is there a rustc bug we could link here?